src/integration/pages/idxPage/persian_afghanistan/canonical.test.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @service persian
+ * @pathname /persian/afghanistan
+ */
+
+import runCrossPlatformTests from '../crossPlatformTests';
+import runCanonicalTests from '../canonicalTests';
+
+describe('Canonical', () => {
+  describe(pageType, () => {
+    runCrossPlatformTests();
+    runCanonicalTests();
+  });
+});
```

Comment:
As above, could we try and get the idx tests into the same format as live radio: https://github.com/bbc/simorgh/blob/idx-integration-tests/src/integration/pages/liveRadio/korean/canonical.test.js

This is something which has recently been refactored (and the intention would be to do this for the rest of the page types). It would mean we wouldn't need to refactor it again :)